docs/examples.rst

```diff
@@ -1,31 +1,116 @@
-   -  `PixelCNN <https://github.com/google/flax/tree/master/linen_examples/linen_examples/pixelcnn/>`__
+   -  `MNIST <https://github.com/google/flax/tree/master/linen_examples/mnist/>`__ [`Interactive
+      <https://colab.research.google.com/github/google/flax/blob/master/linen_examples/mnist/mnist.ipynb>`__] :
+      Convolutional neural network for MNIST classification (featuring simple code).
+   -  `ImageNet <https://github.com/google/flax/tree/master/linen_examples/imagenet/>`__ :
+      Resnet-50 on imagenet with weight decay (featuring multi host SPMD, custom
+      preprocessing, checkpointing, dynamic scaling, mixed precision).
 
 Reinforcement Learning
 
    -  `Proximal Policy
-      Optimization <https://github.com/google/flax/tree/master/linen_examples/linen_examples/ppo/>`__
+      Optimization <https://github.com/google/flax/tree/master/linen_examples/ppo/>`__ :
+      Learning to play Atari games (featuring single host SPMD, RL setup).
 
 Natural language processing
 
    -  `Sequence to sequence for number
-      addition <https://github.com/google/flax/tree/master/linen_examples/linen_examples/seq2seq/>`__
+      addition <https://github.com/google/flax/tree/master/linen_examples/seq2seq/>`__
+      (featuring simple code, LSTM state handling, on the fly data generation).
    -  `Transformer model on
-      WMT <https://github.com/google/flax/tree/master/linen_examples/linen_examples/wmt/>`__
+      WMT <https://github.com/google/flax/tree/master/linen_examples/wmt/>`__ :
+      Translating English/German (featuring multihost SPMD, dynamic bucketing, attention cache,
+      packed sequences, recipe for TPU training on GCP).
 
 Generative models
 
    -  `Variational
-      auto-encoder <https://github.com/google/flax/tree/master/linen_examples/linen_examples/vae/>`__
\ No newline at end of file
+      auto-encoder <https://github.com/google/flax/tree/master/linen_examples/vae/>`__ :
+      Trained on binarized MNIST (featuring simple code, vmap).
+   -  `PixelCNN++ <https://github.com/google/flax/tree/master/linen_examples/pixelcnn/>`__ :
+      Trained on cifar10 (featuring single host SPMD, checkpointing, Polyak decay).
+
+
+Flax examples from the community
+--------------------------------
+
+In addition to the curated list of official Flax examples, there is a growing
+community of people using Flax to build new types of machine learning models. We
+are happy to showcase any example built by the community here! If you want to
+submit your own example, we suggest that you start by forking one of the
+official Flax example, and start from there.
+
+Unfortunately we don't have any examples from the community using the new Linen
+API yet...
+
++--------------------------------+--------+-----------+-----------+
+|              Link              | Author | Task type | Reference |
++================================+========+===========+===========+
+| (waiting for first submission) |        |           |           |
```

Comment:
We can add Hugging Face, https://github.com/rwightman/efficientnet-jax, and I believe that @marcvanzee is porting https://github.com/nikitakit/flax_bert to Linen as well?

Shall we do this in a susbetquent commit?